app/components/windows/ChildWindow.vue

```diff
@@ -1,12 +1,46 @@
 <template>
-<div style="height: 100%">
-    <!--
-        we use 2 components for situation when we need to open a new window and
-        save the previous window-component state
-    -->
-    <component v-if="components[0].name" v-show="components[0].isShown" :is="components[0].name"/>
-    <component v-if="components[1].name" v-show="components[1].isShown" :is="components[1].name"/>
+<div style="height: 100%" :class="{'night-theme': nightTheme, 'day-theme': !nightTheme}">
```

Comment:
is there a reason we set the theme explicitly here?

Reply:
It's needed by the blank slate, which is above the ModalLayout (where we currently set it) in the hierarchy.  I tried removing it from the ModalLayout after this change, but it broke a ton of stuff and I'm not entirely sure why at the moment.